src/VideoStreaming/VideoStreaming.pri

```diff
@@ -15,7 +15,7 @@ LinuxBuild {
     QT += x11extras waylandclient
     CONFIG += link_pkgconfig
     packagesExist(gstreamer-1.0) {
-        PKGCONFIG   += gstreamer-1.0  gstreamer-video-1.0 gstreamer-gl-1.0
+        PKGCONFIG   += gstreamer-1.0  gstreamer-video-1.0 gstreamer-gl-1.0 egl
```

Comment:
The does also require a change to AppImage creation?

Reply:
I don't think so - EGL should be system supplied library because it depends on hardware.

The problem here that EGL dependency should be added during gstreamer-gl-1.0 package resolving. IMO we should not have EGL in our dependency list because we don't use it directly. But since it does not work as need in case of parallels we're adding EGL directly. I'm not sure if EGL is always present on end user systems. Can we test that somehow?

Reply:
I'll need to push it through. Wait for builds to get kicked out. Create a virgin Ubuntu VM and test.

Reply:
Tested and it's fine